canonical-web-and-design/jaas-dashboard
# Add check for controllerData before accessing keys

## Done

Add check for controllerData before accessing keys

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/controllers
- Verify no errors

## Details

Fixes #758
